### PR TITLE
Bring style

### PR DESCRIPTION
**Pull request by maxbachmann**

### CODE REVIEW

File: PublishedModules/philipp2310/BringShoppingList/BringShoppingList.py [hunk 1]
```diff
@@ -168,62 +178,61 @@ def deleteItem(self, session, intent) -> bool:
...
 
 
 	#### List/Text operations
-	### Combines two lists(if filled)
-	# first+CONN+second
-	# first
-	# second
 	# noinspection PyUnusedLocal
-	def _combineLists(self, str_first: str, str_conn: str, str_end: str, str_second: str, first: list, second: list) -> str:
+	def _combineLists(self, strFirst: str, strConn: str, strEnd: str, strSecond: str, first: list, second: list) -> str:
```
Comment (maxbachmann) [inline PublishedModules/philipp2310/BringShoppingList/BringShoppingList.py]: why do strConn and strEnd exist when they are never used?

File: PublishedModules/philipp2310/BringShoppingList/BringShoppingList.py [hunk 2]
```diff
@@ -55,36 +56,41 @@ def onMessage(self, intent: str, session: DialogSession) -> bool:
...
 			else:
 				self.endDialog(session.sessionId, text=self.randomTalk('nodel_all'))
 
 		return True
 
 
-	# get an instance of the BringApi
 	def _getBring(self) -> BringApi:
+		"""get an instance of the BringApi"""
 		return BringApi(self._uuid, self._uuidlist)
```
Comment (maxbachmann) [inline PublishedModules/philipp2310/BringShoppingList/BringShoppingList.py:78]: what happens when there is not internet or authentification stuff fails?

**Comment by maxbachmann** [comment]
in the training example there are many values that appear broken. e.g. 
```
{
    "value": "MondelÄ“z International"
},
```

**Comment by maxbachmann** [comment]
Instead of creating your own yes/no intent **confDelList_bringshop** you should probably use the intent **AnswerYesOrNo**. But even when you decide that you would like to create a own one (that just has a different name, but does the same) it should not be enabled by default like it is right now, since you only need it activated when you ask the user for confirmation

**Comment by Psychokiller1888** [comment]
Non utf-8 characters?

**Comment by Psychokiller1888** [comment]
Re Yes / No, yep, should use the default one, better, easier :)

File: PublishedModules/philipp2310/BringShoppingList/BringShoppingList.py (hunk 1 above)
Comment (maxbachmann) [reply PublishedModules/philipp2310/BringShoppingList/BringShoppingList.py]: @philipp2310 confirmed that they are deprecated

**Pull request merged**